satellite/repair/checker/checker.go

```diff
@@ -244,6 +244,8 @@ func (obs *checkerObserver) RemoteSegment(ctx context.Context, path metainfo.Sco
 	mon.IntVal("checker_segment_total_count").Observe(int64(len(pieces)))
 	mon.IntVal("checker_segment_healthy_count").Observe(int64(numHealthy))
 
+	mon.IntVal("checker_segment_age").Observe(int64(time.Since(pointer.CreationDate)))
```

Comment:
convert to seconds - probably better to split this onto two lines:
`segmentAge := time.Since(pointer.CreationDate)`
`mon.IntVal...Observe(int64(segmentAge.Seconds()))`